site/src/contexts/LocalPreferencesContext.tsx

```diff
@@ -45,7 +45,7 @@ export const LocalPreferencesProvider = ({
     if (preferencesStr) {
       try {
         const values = JSON.parse(preferencesStr)
-        setState({ ready: true, values })
+        setState({ ready: true, ...values })
```

Comment:
Should it be the other way around?  Otherwise if the values contain `ready: false` that would take precedence over the true.

Reply:
This should not happen but I will put more thoughts on this... it is not looking great to be honest.